docs/5.x/changelog.md

```diff
@@ -125,12 +125,20 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.0.34 LTS
+
+### Bugfixes
+* Fix security issue with Kapacitor service.
```

Comment:
If there was a security issue, should these notes include a warning similar to the security releases below? cc @knisbet (and maybe give more info about what the issue was?)

Reply:
I agree with Roman

Reply:
Can you take a look at the new sentence?

Reply:
looks better to me.